client/galaxy/style/less/base.less

```diff
@@ -1719,3 +1719,23 @@ div.toolTitleNoSection
 #for_bears {
     display: none;
 }
+
+// communication channel bootstrap modal 
```

Comment:
Yes, please create a new directory in `config/plugins`, maybe `tabs` or `menu` and move all needed files to there.